tests/queries/0_stateless/2021_exponential_sum.reference

```diff
@@ -0,0 +1,8 @@
+0.0009765625
+0.0009775171065493646
+0.0009775171065493646
+0.0009775171065493646
+0.0009775171065493646
+0.0009775171065493646
+0.0009775171065493646
+0.0009775171065493646
```

Comment:
It is not clear to me why this value is always the same, reading from _mt storage.

Reply:
Now calculation of moving average made that way, so it does not depend on order of values.
(Only depends on what will be the latest value by time)